.gitmodules

```diff
@@ -5,4 +5,6 @@
 [submodule "regression-tests"]
 	path = regression-tests
 	url = https://github.com/pulp-platform/regression_tests.git
-    branch = astral
```

Comment:
Are we sure we want to remove the branch name?

Reply:
This might be an unwanted artifact of handling the submodules during a rebase. I will add it back.